packages/plugin-minify/package.json

```diff
@@ -14,15 +14,18 @@
   "author": "Andrew Powell <andrew@shellscape.org>",
   "homepage": "https://jsx.email/",
   "main": "./dist/commonjs/index.js",
+  "module": "./dist/esm/index.js",
```

Comment:
you are never permitted to change package.json files per your instructions in `.charlie`